package.json

```diff
@@ -22,13 +22,6 @@
       "import": "./dist/index.js"
     }
   },
-  "typesVersions": {
-    "*": {
-      "*": [
-        "./dist/*"
```

Comment:
let's add `"./*""` to the array instead of removing the values. That should also do for older versions.